### PR TITLE
Add option for having animations

### PR DESCRIPTION
**Pull request by KomalBhadani27**
**Command to run from CLI:**
`$python3 main.py 'https://www.scoopwhoop.com/news/no-women-in-aap-cabinet/?ref=colllection_list' -enable_animations 'on'
`

**On web interface:**
Image

### CODE REVIEW

File: stampifier.py [hunk 1]
```diff
@@ -20,12 +20,13 @@
...
     """Creates class for stampification"""
 
-    def __init__(self, url, max_pages):
+    def __init__(self, url, max_pages, add_animations):
         self.url = url
         self.max_pages = max_pages
         self._website = Website(self.url)
         self.is_stampifiable = False
         self.stampified_pages = None
+        self.add_animations = add_animations
```
Comment (detel) [inline stampifier.py:29]:
```suggestion
        self.enable_animations = enable_animations
```

File: api/server.py [hunk 2]
```diff
@@ -25,12 +25,16 @@ def home():
...
+    animations = False
     if 'website_url' in request.args:
         url = request.args.get('website_url')
     if 'max_pages' in request.args:
         max_pages = request.args.get('max_pages')
+    if 'animations' in request.args:
+        animations = request.args.get('animations')
 
-    stampified_url = '/stampified_url?url=%s&max_pages=%s' % (url, max_pages)\
+    stampified_url = '/stampified_url?url=%s&max_pages=%s&animations=%s' % \
```
Comment (detel) [inline api/server.py:36]: Use [urllib](https://docs.python.org/3/library/urllib.parse.html) to build URLs.

Comment (KomalBhadani27) [reply api/server.py]: Thanks for the suggestion! Updated!

Comment (detel) [reply api/server.py]: https://stackoverflow.com/questions/7801991/how-to-convert-a-dictionary-to-query-string-in-python

Comment (KomalBhadani27) [reply api/server.py]: Will look into it. Missed this earlier. Thanks for the link.

**Pull request merged**